### PR TITLE
Bug 1910372: Backport redistribute new root creds in passthrough mode

### PR DESCRIPTION
**Pull request by akhil-rane**
Backporting [this PR](https://github.com/openshift/cloud-credential-operator/pull/242) to 4.6 release

jira: https://issues.redhat.com/browse/OCPBUGSM-20704

### CODE REVIEW

File: pkg/aws/actuator/actuator.go [hunk 1]
```diff
@@ -227,9 +227,23 @@ func (a *AWSActuator) needsUpdate(ctx context.Context, cr *minterv1.CredentialsR
 
 	} else {
 		// for passthrough creds, just see if we have the permissions requested in the credentialsrequest
```
Comment (joelddiaz) [inline pkg/aws/actuator/actuator.go:229]: In the original PR, this line was removed. Any reason this PR isn't a cherry pick of #242

Comment (akhil-rane) [reply pkg/aws/actuator/actuator.go:229]: Yes, looks like cherry pick is a better option here. I thought some changes won't go in 4.6, but I was wrong. I will update the PR. Thanks!

Comment (akhil-rane) [reply pkg/aws/actuator/actuator.go]: Removed this line.

Comment (akhil-rane) [reply pkg/aws/actuator/actuator.go]: lets go ahead with this PR as automated cherry pick of #242 [resulted in conflicts](https://github.com/openshift/cloud-credential-operator/pull/242#issuecomment-749090096). Removed the above line.

**Review by joelddiaz — COMMENTED** [review]
/lgtm

**Comment by akhil-rane** [comment]
@joelddiaz looks like this is blocked on cherry-pick-approved label. Never heard of it before.

**Comment by joelddiaz** [comment]
@akhil-rane yes, the patch manager has to individually approve any PR before it can slip into a z-stream release. The patch manager changes per z-stream, so I can't give you a single person to talk to, but this PR should be on their radar when considering which PRs to accept for a given z-stream release.

**Pull request merged**